CollapseLauncher/Classes/GameManagement/GameSettings/Genshin/RegistryClass/GeneralData.cs

```diff
@@ -174,104 +200,124 @@ internal class GeneralData
 
         /// <summary>
-        /// This define "<c>Text Language</c>" in-game settings. <br/>
-        /// Valid values: 1-13 
-        /// Default: 1
-        /// English(1)
-        /// Simplified Chinese(2)
-        /// Traditional Chinese(3)
-        /// French(4)
-        /// German(5)
-        /// Spanish(6)
-        /// Portugese(7)
-        /// Russian(8)
-        /// Japanese(9)
-        /// Korean(10)
-        /// Thai(11)
-        /// Vietnamese(12)
-        /// Indonesian(13)
-        /// Turkish(14)
-        /// Italy(15)
+        /// This define vibration level for certain controller probably ?
+        /// Valid Values: 0 (Full), 1 (Partial), 2 (Off)
+        /// Default: 0
         /// </summary>
-        public int deviceLanguageType { get; set; } = 1;
-
-        public List<string> _customDataKeyList { get; set; }
-        public List<object> mtrBanReasons { get; set; }
-        public int mtrTimeInterval { get; set; } = 1000;
+        public int vibrationLevel { get; set; } = 0;
 
         /// <summary>
-        /// This define "<c>SFX Volume</c>" slider in-game.<br/>
-        /// Valid values: 0-10
-        /// Default: 10
+        /// This defines "<c>Vibration Intensity</c>" slider in-game. <br/>
+        /// Valid Values: 1-5
+        /// Default: 5
         /// </summary>
-        public int volumeSFX { get; set; } = 10;
-
-        public int mtrAutoTraceInterval { get; set; } = 3600;
-        public bool needReportQuestResourceDeleteStatusFiles { get; set; } = false;
+        public int vibrationIntensity { get; set; } = 5;
 
         /// <summary>
-        /// This holds said override for those controllers.
+        /// Some vibration related stuff ?
         /// </summary>
-        // Temporary for fallback before the implementation is made
-        public List<string> _overrideControllerMapValueList { get; set; }
-
-        public int mtrTimeOut { get; set; } = 5000;
+        public bool usingNewVibrationSetting { get; set; } = true;
 
         /// <summary>
-        /// This define "<c>Global Volume</c>" slider in-game.<br/>
-        /// Valid values: 0-10
-        /// Default: 10
+        /// Is not an actual blur setting
         /// </summary>
-        public int volumeGlobal { get; set; } = 10;
+        public bool motionBlur { get; set; } = true;
 
+        /// <summary>
+        /// Gyro Aiming stuff, most likely not used in PC.
+        /// </summary>
+        public bool gyroAiming { get; set; } = false;
+        public int gyroHorMoveSpeedIndex { get; set; } = 2;
+        public int gyroVerMoveSpeedIndex { get; set; } = 2;
+        public bool gyroHorReverse { get; set; } = false;
+        public bool gyroVerReverse { get; set; } = false;
+        public int gyroRotateType { get; set; } = 0;
+        public bool gyroExcludeRightStickVerInput { get; set; } = false;
+
+        //unsure what these does, probably HDR stuff? doesn't have HDR monitor to test...
```

Comment:
This looks like it controls HDR, maybe check Unity to see if it has these parameters? We should try and map this to something, or comment it if we're not planning on using them.

Reply:
No in-game settings mapped to this so I'm not sure if we want to mess with it

Reply:
Made it clearer as of https://github.com/neon-nyan/Collapse/commit/f2f38dd39f11b69fdc06ac1e2bca0530f6571b40